src/modules/auth/auth.service.ts

```diff
@@ -119,13 +119,18 @@ export class AuthService {
       enabled: 'true',
       firstName: body?.firstName,
       lastName: body?.lastName,
-      username: body?.phoneNumber,
+      username:
+        body?.firstName +
```

Comment:
@GitVivekHub could you please use a function to generate the username and consider the extra spaces in data, you can use trim or replace it with an underscore 
**Ref** below function  and as discussed yesterday make replace aadhaar with the phone number in this function too
https://github.com/PSMRI/beneficiary-backend/blob/main/src/modules/users/users.service.ts#L704